app/src/androidTest/java/kommunicate/io/sample/utils/Utils.kt

```diff
@@ -0,0 +1,54 @@
+package kommunicate.io.sample.utils
+
+import android.util.Base64
+import android.view.View
+import androidx.test.espresso.UiController
+import androidx.test.espresso.ViewAction
+import androidx.test.espresso.matcher.ViewMatchers.isRoot
+import com.google.gson.JsonParser
+import kommunicate.io.sample.data.RequestTokenData
+import kommunicate.io.sample.network.KommunicateDashboardAPI
+import kotlinx.coroutines.runBlocking
+import org.hamcrest.Matcher
+import org.junit.Assert.fail
+
+/**
+ * chatAuth = 0
+ * dashboardAuth = 1
+ * */
+fun getAuthToken(dashboardAPI: KommunicateDashboardAPI): List<String> {
+    val token = mutableListOf<String>()
+    runBlocking {
+        val tokenData = RequestTokenData(
+            true,
+            "JTdCJTIydXNlck5hbWUlMjIlM0ElMjJwcmF0ZWVrLnNpbmdoJTQwa29tbXVuaWNhdGUuaW8lMjIlMkMlMjJwYXNzd29yZCUyMiUzQSUyMnByYXRlZWslMjQlMjIlMkMlMjJhcHBsaWNhdGlvbk5hbWUlMjIlM0ElMjIlMjIlMkMlMjJhcHBsaWNhdGlvbklkJTIyJTNBJTIyZDZjYmMyMzIyYzYwODUxOWFkNjVhYjNiY2IwOWZlNzglMjIlMkMlMjJkZXZpY2VUeXBlJTIyJTNBMCU3RA=="
```

Comment:
Is this a JWT token? What happen if this expires

Reply:
No it's not, Its Base 64 encoded string of my creds. So, It will be always same.